playbook-swift/readME.md

```diff
@@ -0,0 +1,100 @@
+# Playbook for Swift 
+### Table of Contents
+	* [Folder Structure](bear://x-callback-url/open-note?id=05A5108D-03BF-4652-B3EE-33A2ABC7DF4C-497-0000E555E8E0C1FB&header=Folder%20Structure)
```

Comment:
what are we doing for people that don't have Bear?

Reply:
@jasperfurniss that needs to just be removed